extension/app/ts/simulation/services/SimulationModeEthereumClientService.ts

```diff
@@ -326,17 +326,11 @@ export const getSimulatedCode = async (ethereumClientService: EthereumClientServ
 	const getCodeTransaction = {
 		type: '1559' as const,
 		from: MOCK_ADDRESS,
-		chainId: await ethereumClientService.getChainId(),
+		chainId: ethereumClientService.getChainId(),
 		nonce: await ethereumClientService.getTransactionCount(MOCK_ADDRESS),
 		maxFeePerGas: 0n,
 		maxPriorityFeePerGas: 0n,
-		gas: await simulateEstimateGas(ethereumClientService, simulationState, {
-			from: MOCK_ADDRESS,
-			to: GET_CODE_CONTRACT,
-			data: input,
-			value: 0n,
-			gasPrice: 0n,
-		}),
+		gas: 94104n,
```

Comment:
Gas costs can change.  Can we perhaps memoize this value instead of hard coding it so we don't have something randomly break in a future hardfork?

Reply:
you mean its called once and then stored? When do we know when we should update it again?

Reply:
Correct.  Just once on startup is probably fine, if it breaks the user can just "turn it off and on again".

Reply:
hmm, I would rather not add a lot stuff at the startup. Maybe it could be done first time its called and stored